docs/examples.md

```diff
@@ -514,6 +513,25 @@ gcloud functions deploy local-nuget \
   --entry-point Google.Cloud.Functions.Examples.LocalNuGetPackageFunction.Function
 ```
 
+## Typed GoogleChatBot
```

Comment:
The section title should reflect the project name, i.e. GoogleChatBot. (That's the convention for the whole of this document.)